Files.Launcher/Files.Launcher.csproj

```diff
@@ -159,10 +159,10 @@
       <Version>4.7.0</Version>
     </PackageReference>
     <PackageReference Include="Vanara.PInvoke.Mpr">
-      <Version>3.3.7</Version>
+      <Version>3.3.8</Version>
```

Comment:
There are some breaking changes with this update, have you confirmed that everything is working correctly?

Reply:
So far it seems to work just fine, I will let you know if there's any issue caused by the new version